client/src/app/society/[society]/page.tsx

```diff
@@ -0,0 +1,112 @@
+import { MdEmail } from "react-icons/md";
+import { FaFacebook } from "react-icons/fa";
+import { AiOutlineGlobal } from "react-icons/ai";
+import Image from "next/image";
+import Chip from "@/components/Chip";
+import { createRipple } from "@/components/Button";
+import { SquarePen } from "lucide-react";
+
+export default function SocietyPage() {
+  const Top3tags = ["test", "test", "test"];
+  const avgStar = 4.5;
+  const percentage = ((avgStar / 5) * 100).toFixed(1) + "%";
+  const reviews = 12;
+
+  return (
+    <>
+      <Image
+        priority
+        src={"/assets/wave.svg"}
+        alt="Green Wave"
+        className="pointer-events-none select-none"
+        sizes="100vw"
+        width={0}
+        height={0}
+        style={{ width: "100%", height: "auto" }}
+      />
+      <div className="grid grid-cols-2 w-full px-20 gap-40">
+        <div className="flex flex-col sticky top-0 pt-10 rounded-lg max-h-screen overflow-auto">
+          <div className="flex flex-row gap-5">
+            <div className="flex flex-col gap-3">
+              <Image
+                priority
+                src={"https://cdn.linkupevents.com/arc_logo.png"}
+                alt="Society Logo"
+                className="rounded-full"
+                sizes="100vw"
+                width={100}
+                height={100}
+              />
+              <div className="flex justify-center items-center gap-2">
```

Comment:
Also consider what happens when there are 5 logos: email, facebook, website, discord, instagram - how would you want it to look like?